package.json

```diff
@@ -48,7 +49,7 @@
     "build": "react-app-rewired build",
     "test": "react-app-rewired test --passWithNoTests",
     "cypress": "npx cypress run",
-    "lint": "eslint --fix './src/**/*.{js,ts,tsx}'",
+    "lint": "eslint --fix ./src/**/*.{js,ts,tsx}",
```

Comment:
why'd you remove the '? afaik the glob pattern wasn't working without it

Reply:
npm run lint didn't run for me with the ' on : (

Reply:
I had to remove the ' for the command to work

Reply:
i had to add ' for the command to work -_______________________-

Reply:
@thaigillespie can you retest this with wsl? ' should work

Reply:
Yeah I'll test it again later. ' didn't work in windows but should work in wsl.

Reply:
sounds good - thanks!